### PR TITLE
Implemented caching

### PR DESCRIPTION
**Pull request by EricLarson2020**
Implemented caching in views

### CODE REVIEW

File: app/views/application/_browsing_index.html.erb [hunk 1]
```diff
@@ -1,15 +1,17 @@
 <div class="container">
-<% @obituaries.each do |obit| %>
-  <div class="row" style= "background-color:white;">
-    <div class="col-md-3 p-4">
-      <%= link_to image_tag(obit.image_url), "/obituaries/#{obit.id}" %>
+<% cache(action: "index", action_suffix: "obituaries_index") do %>
```
Comment (EricLarson2020) [inline app/views/application/_browsing_index.html.erb:2]: Just added this in all views, there are a few other ways to do it. The action suffix distinguishes between different caching on the page, the index is just the action

File: config/environments/development.rb [hunk 2]
```diff
@@ -13,18 +13,14 @@
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
-  if Rails.root.join('tmp/caching-dev.txt').exist?
```
Comment (EricLarson2020) [inline config/environments/development.rb:16]: This turns caching on

File: spec/services/obituary_service_spec.rb [hunk 3]
```diff
@@ -21,4 +21,36 @@
 
     expect(obituaries.first[:data][:attributes][:abstract].downcase.include?('coronavirus') || obituaries.first[:data][:attributes][:abstract].downcase.include?('covid')).to eq(true)
   end
+
+  it "returns recent obituaries", :vcr do
```
Comment (EricLarson2020) [inline spec/services/obituary_service_spec.rb:25]: Added more tests for service

File: config/environments/development.rb (hunk 2 above)
Comment (stellakunzang) [reply config/environments/development.rb:16]: Ooo nice, so there is built in caching code we just have to implement?

File: spec/services/obituary_service_spec.rb (hunk 3 above)
Comment (stellakunzang) [reply spec/services/obituary_service_spec.rb:25]: Thank you, this is awesome! Appreciate your thorough testing :grateful:

File: config/environments/development.rb (hunk 2 above)
Comment (EricLarson2020) [reply config/environments/development.rb:16]: Yep, rails is hookup up to do caching, we just have to turn off the blocker by deleting a few lines of code

**Pull request merged**